test/test_dos.py

```diff
@@ -5219,6 +5219,7 @@ def setUpClass(cls):
             "test_fat_label_create_noduplicate": KNOWNFAIL,
             "test_fat_label_create_predir": KNOWNFAIL,
             "test_fat_label_create_prefile": KNOWNFAIL,
+            "test_lfs_disk_info_mfs": KNOWNFAIL,
```

Comment:
Why though?

Reply:
FreeDOS 1.3 doesn't use the extension we added here https://github.com/FDOS/kernel/commit/fa3dc593672604e479f49b3fa10f494df3e7cb2a, the next released kernel will be okay.